tools/update_to_relative_path/MakeArchiveLocationRelative.pl

```diff
@@ -0,0 +1,218 @@
+#! /usr/bin/perl
```

Comment:
@cmadjar 

I agree the script should me boved to tools/ but why create a sub-directory inside the tools/ directory that is super specific?

Reply:
hehe, boved, do you have a cold? ;)

Because there is a second one that will go in the same directory. Two scripts were created to make the paths relative in the DB, one on the dicom-archive side and one on the LORIS-MRI side. The other is in PR #243

Reply:
> @MounaSafiHarab hehe, boved, do you have a cold? ;)

can't believe you saw this; I fixed it in a second, yet you saw old typo!!!

re. path, I still think it might be an overkill to create a sub-directory for every script or couple of scripts; especially these were done to accommodate a release change. But am open to whatever the group decides!

Reply:
Sounds good! How about we mention it during our next meeting and I make the change on whatever we decide. I am not married to any options. BTW, in the redmine ticket, I marked all the files that should probably go into the tools directory. Need more investigation though, but it is a lot of files!

Reply:
sure, already added an item to the meeting agenda; and yes many can go to the tools/ directory and the LDZ one should be removed I think :)

Reply:
Wouldn't it make more sense to just have a single directory for all one time release update scripts?

Reply:
ooooh, very good point! I guess we'll have a lot of pre-19.0 tools cause I believe this is the case for most of them :P.